### PR TITLE
RHISOS 470 | IOPS Values from Rules are actuals

### PR DESCRIPTION
**Pull request by saltgen**
Changes are based on the assumption that IOPS values are actuals from rules

### CODE REVIEW

**Review by PreetiW — COMMENTED** [review]
@saltgen will these changes handle the sorting as well? I haven't tested it yet but I remember last time we had faced it

File: ros/api/v1/hosts.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def build_sort_expression(self, order_how, order_method):
         if order_method in score_methods:
             return (
                 sort_order(PerformanceProfile.performance_utilization[
-                    order_method].astext.cast(Integer)),
+                    order_method].astext.cast(Float)),
                 asc(PerformanceProfile.system_id),)
```
Comment (saltgen) [inline ros/api/v1/hosts.py:203]: @PreetiW Change on line 202, will handle the previous sort error, tested :white_check_mark:

**Review by apuntamb — APPROVED** [review]
Codewise looks ready to go!

**Comment by apuntamb** [comment]
Thanks @saltgen for the quick fix and @PreetiW for testing it sooner! Merged!

**Comment by PreetiW** [comment]
Working like a charm, thanks @saltgen 
image

**Pull request merged**